lab_4_retrieval_w_clustering/main.py

```diff
@@ -287,6 +400,27 @@ def run_single_train_iteration(self) -> list[ClusterDTO]:
         Returns:
             list[ClusterDTO]: List of clusters.
         """
+        used_centroids = []
+        for cluster in self.__clusters:
+            cluster.erase_indices()
+            used_centroid = cluster.get_centroid()
+            used_centroids.append(used_centroid)
+        used_vectors = self._db.get_vectors()
+        for vector in used_vectors:
+            vector_distances = []
+            for centroid in used_centroids:
+                distance = calculate_distance(vector[1],centroid)
```

Comment:
add whitespaces everywhere or try to run black check :)

Reply:
will run black check closer to the deadline

Reply:
great!